### DISCUSSION TITLE
The client does not catch events UserUpdate()

### DISCUSSION THREAD

**Discussion by Freqnsy** [aiogram/aiogram#1341] [Q&A]
The client does not catch events UserUpdate() for some users
```
@client.on(events.UserUpdate())
async def normal_handler(event):
# For some users, events are not caught here (online status)
...
```
What I found out:
- These users are in the account's address book. 
- I can get an ID using both the phone number and the username.
- On different clients (aiogram) all the same
- Even on mobile and desktop applications, the online status of these users is not updated until you close/open the chat.

This is not a problem with **aiogram**, The problem is somewhere higher.
There is an assumption that something is wrong with the ID. Perhaps the reason is deleting the user and re-registering.
Or in some user settings.

Do you have any ideas?

**Comment by JrooTJunior** [comment]
aiogram is missing in the given code fragment, so you have applied to the wrong address.

You should move this question to the resoursec related to library that you are using.
